### ISSUE TITLE
seurat_obj.rds not found while running downstream analysis 

### ISSUE THREAD

**Issue by khushi0810** [wbaopaul/scATAC-pro#68]
Hello Wenbao @wbaopaul ,

I am using v1.5.1 on one of my samples , and I'm using scATAC-pro reprocess because I have my bam files from cell ranger and then using scATAC-pro downstream . 

I am running on HPC as a batch job. I am attaching my log file , configure file as well as my script for you reference.

Any suggestions are highly appreciated.

Thank you
Khushi
[reprocess_cellranger_output.log](https://github.com/wbaopaul/scATAC-pro/files/14501527/reprocess_cellranger_output.log)
[configure_user.txt](https://github.com/wbaopaul/scATAC-pro/files/14501534/configure_user.txt)

#####################################

module load singularity/3.9.9

cd /project/bioinformatics/JLee_lab/s227606/scATAC/NONP

singularity exec --bind /project/bioinformatics/JLee_lab/s227606/scATAC -H /project/bioinformatics/JLee_lab/s227606/scATAC/NONP --cleanenv /project/bioinformatics/JLee_lab/s227606/scATAC/NONP/scatac-pro_latest.sif scATAC-pro -s reprocess_cellranger_output -i /project/bioinformatics/JLee_lab/s227606/scATAC/files/BL51_NP/outs/possorted_bam.bam,/project/bioinformatics/JLee_lab/s227606/scATAC/files/BL51_NP/outs/fragments.tsv.gz -c /project/bioinformatics/JLee_lab/s227606/scATAC/configure_user.txt

singularity exec --bind /project/bioinformatics/JLee_lab/s227606/scATAC -H /project/bioinformatics/JLee_lab/s227606/scATAC/NONP --cleanenv /project/bioinformatics/JLee_lab/s227606/scATAC/NONP/scatac-pro_latest.sif scATAC-pro -s downstream -i /project/bioinformatics/JLee_lab/s227606/scATAC/NONP/output/filtered_matrix/MACS2/FILTER/matrix.mtx -c /project/bioinformatics/JLee_lab/s227606/scATAC/configure_user.txt

###################################################

**Comment by wbaopaul** [comment]
Can you check some intermediate results? Such as how many cells did you get under filtered_Matrix folder or any other output? I guess some of your filtering parameters may be too stringent, like min_frac_peak set as 0.5.

**Comment by khushi0810** [comment]
Hello,

Thank you so much for your reply. I am attaching my results under filtered_matrix folder.
Should I try increasing the threshhold ?

Thanks 
[barcodes.txt](https://github.com/wbaopaul/scATAC-pro/files/14502073/barcodes.txt)
[features.txt](https://github.com/wbaopaul/scATAC-pro/files/14502074/features.txt)

**Comment by wbaopaul** [comment]
No, you need to decrease your thresholds, for instance, min_uniq_frags to 2000, min_frac_peak 0.25.  You can increase max_frac_mito to 0.15 or 0.2.

**Comment by khushi0810** [comment]
Thank you so much. I changed the threshholds and was able to resolve this error but bumped into another error 

Error in FindNeighbors.Seurat(seurat.obj, reduction = "pca", dims = 1:30,  : 
  More dimensions specified in dims than have been computed
Calls: FindNeighbors -> FindNeighbors.Seurat
Execution halted

Should I change the nReduction parameter ?

Error: Cannot find 'active_clusters' in this Seurat object
Execution halted
Report generation Done!

Do I need to change this parameter ?
K_CLUSTERS = 0.2  ## the number of cluster (in integer) or the resolution parameter (in float) for louvain algorithm (implemented by seurat) 

Thank you !

**Comment by wbaopaul** [comment]
Not sure what exactly happened. Have you checked the outputs? I guess you probably ended up too few number of cells.
